cmd/controller-manager/app/options/federatedResourceQuotaEnforcement.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// FederatedResourceQuotaEnforcementOptions holds the FederatedResourceQuotaEnforcement options.
+type FederatedResourceQuotaEnforcementOptions struct {
```

Comment:
```suggestion
type FederatedResourceQuotaOptions struct {
```
It should take all FRQ-related options.